src/crewai/crew.py

```diff
@@ -520,6 +531,9 @@ def kickoff(
                 f"The process '{self.process}' is not implemented yet."
             )
 
+        for callback in self.after_kickoff_callbacks:
+            result = callback(result)
```

Comment:
qq: In case one of the callbacks raises an exception, will it stop the whole process?

Reply:
Good question. The process will break in this case, and I believe it's a good thing, as we can't silently move on if something went wrong in the process of changing inputs.